src/models/focus.js

```diff
@@ -99,9 +100,19 @@ nv.models.focus = function(content) {
             brush
                 .x(x)
                 .on('brush', function() {
-                    onBrush();
+                    onBrush(syncBrushing);
                 });
 
+            if (!syncBrushing) {
+                brush.on('brushend', function () {
+                    var extent = brush.empty() ? x.domain() : brush.extent();
+                    if (Math.abs(extent[0] - extent[1]) <= 1) {
+                        return;
+                    }
+                    dispatch.onBrush(extent);
```

Comment:
oh whoops, I got the dispatch and the regular onBrush function confused there, my bad.